proto/ibc/core/client/v1/query.proto

```diff
@@ -128,3 +133,20 @@ message QueryClientParamsResponse {
   // params defines the parameters of the module.
   Params params = 1;
 }
+
+// QueryUpgradedClientStateRequest is the request type for the Query/UpgradedClientState RPC
+// method
+message QueryUpgradedClientStateRequest {
+  // client state unique identifier
+  string client_id = 1;
+  // plan height of the current chain must be sent in request
+  // as this is the height under which upgraded client state is stored
+  int64 plan_height = 2;
+}
+
+// QueryUpgradedClientStateResponse is the response type for the Query/UpgradedClientState RPC
+// method.
+message QueryUpgradedClientStateResponse {
+  // client state associated with the request identifier
+  google.protobuf.Any upgraded_client_state = 1;
```

Comment:
We can add proof slots later, but the relayer constructs its own proof via an ABCI query so I wanted to save time